test/e2e/scalability/density.go

```diff
@@ -901,13 +899,8 @@ func createRunningPodFromRC(wg *sync.WaitGroup, c clientset.Interface, name, ns,
 			},
 		},
 	}
-	for attempt := 1; attempt <= MaxLatencyPodCreationTries; attempt++ {
-		_, err := c.CoreV1().ReplicationControllers(ns).Create(rc)
-		if err == nil || apierrs.IsAlreadyExists(err) {
-			break
-		}
-		Expect(attempt < MaxLatencyPodCreationTries && framework.IsRetryableAPIError(err)).To(Equal(true))
-	}
+	err := testutils.CreateRCWithRetries(c, ns, rc)
+	framework.ExpectNoError(err)
```

Comment:
framework.ExpectNoError(testutils.CreateRCWithRetries(c, ns, rc))

Reply:
Done. Actually I made that change when writing the code, but undo it feeling that it looks cleaner the other way :)